nursery/get-current-process-command-line.yml

```diff
@@ -0,0 +1,15 @@
+rule:
+  meta:
+    name: get current process command line
+    # namespace: host-interation/process
```

Comment:
why not include the namespaces?

how is this displayed in the default/v/vv output rendering?

Reply:
```suggestion
    namespace: host-interation/process
```